### PR TITLE
fix: not uncheck filter's checkbox when reset image [fix #552]

### LINKED ISSUE #552: Filters check boxes are still checked on Reset
**Describe the bug**
Reset action only resets the filters on the preview, but filters' check boxes are still checked. So to apply the same filter you should uncheck it and then check it again

**To Reproduce**

1. Go to http://nhn.github.io/tui.image-editor/latest/tutorial-example01-includeUi
2. Click on the Grayscale filter checkbox
3. Click on Reset action

**Expected behavior**
The Grayscale filter checkbox is unchecked

**Screenshots**
The Grayscale filter checkbox is still checked, while the filter itself was cleared on the preview

**Desktop (please complete the following information):**

- OS: Ubuntu 20.04
- Browser Chrome
- Version 88.0.4324.96 (Official Build) (64-bit)

### PR DESCRIPTION
**Pull request by lja1018**
### Please check if the PR fulfills these requirements

- [x] It's submitted to right branch according to our branching model
- [x] It's right issue type on title
- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

### Description

Fix a bug that does not uncheck the filter's check box when a reset or image is loaded after applying the filter

* AS-IS
  * 화면-기록-2021-04-01-오후-5 07 09
* TO-BE
  * Uncheck the filter box when resetting or loading images after applying the filter.

* Solution
  * Add logic to uncheck all filter check boxes when reset or image load
---

Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

**Review by jung-han — APPROVED** [review]
완료입니다

File: apps/image-editor/src/js/ui/filter.js [hunk 1]
```diff
@@ -175,6 +175,19 @@ class Filter extends Submenu {
     this.checkedMap[filterName].checked = !isRemove;
   }
 
+  /**
+   * Init all filter's checkbox to unchecked state
```
Comment (jung-han) [inline apps/image-editor/src/js/ui/filter.js:179]: 이 주석은 없어도 괜찮지 않을까요?_?

Comment (jung-han) [reply apps/image-editor/src/js/ui/filter.js:179]: 추가로 실패 케이스에 대한 테스트가 생성될 수 있으면 생성하면 좋을것 같습니다 ㅎ

**Review by dotaitch — APPROVED** [review]
리뷰 완료합니다.

**Pull request merged**